setup.py

```diff
@@ -70,13 +70,13 @@ def run(self):
     zip_safe=False,
     include_package_data=True,
     setup_requires=[
-        'flake8>=3.8,<4',
+        'flake8>=3.8',
```

Comment:
```suggestion
        'flake8>=3.8,<5',
```

The other version bound changes are likely unwanted.

Reply:
im optimistic that flake8>4 will still work, since libs are mostly backward compatible